### DISCUSSION TITLE
S7-1500 write issues

### DISCUSSION THREAD

**Discussion by rtok95** [fbarresi/Sharp7#30] [Q&A] [answered]
Every so often a write command seems to not be processed by the PLC. Regardless of single write of multi var write.

Only happens to S7-1500. Tried with S7-1200 and this PLC does not experience this problem. Also tried different firmware versions of S7-1500. Wireshark shows the write command, and it is also confirmed by the PLC with a success, however, the actual value in the PLC is not changed.

Any solution to this?

**Comment by rtok95** [comment, marked as answer]
Solution found, I used the data I was writing to as a INOUT variable somewhere in the application...

**Reply by fbarresi** [reply]
And don't forget to ⭐  the project 👍

**Comment by fbarresi** [comment]
Wow! Happy to hear you found the solution in only 30 minutes! 😄 
Feel free to comment below if you still need more help.
Best regards, 
FB
